### ISSUE TITLE
Programs search for students and cannot find them

### ISSUE THREAD

**Issue by thomasnatalie** [SMSK-CO/SMSK#59]
Hi Craig, 

Multiple programs have contacted me to let me know that they search for a student and the student does not appear or says they do not exist. Then, when they go to enter the student, the record will show up. I have asked them to send screen shots as long as they can avoid sending identifying information. If they can't, can I have them send screen shots to you? We are in the middle of figuring out a solution for them to send PHI to me but for the mean time, I cannot receive their screenshots if they have any PHI. 
@mquigley1 - Is this similar to the issue KIND was having?

**Comment by craigmcampbell** [comment]
Sure, they can send info to me. At this point, I probably don't need to see screenshots, but names that aren't working or didn't work. I don't know if it is possible, but it would be great if they let me know when they don't find someone they are expecting before they try to add them, that way I can see what the state of the data is before they get added. Let me know if you'd like to chat about this.

**Comment by thomasnatalie** [comment]
I will have Nancy with Chaffee (she is the most recent person to communicate this issue to me) send the name of the next student she can't find before adding. 

Thanks, Craig!

**Comment by mquigley1** [comment]
Hi, 
Yes,, this seems to be the same issue KIND was hsving.
Michelle 

Sent from my T-Mobile 4G LTE Device

-------- Original message --------

**Comment by craigmcampbell** [comment]
I've been digging into this, but I'm just not seeing any issues. I pick random students and I'm able to search for them and they come up in the results fine. 

Not long before launching the new code, student search was changed. I'm wondering if they are using the search in an unexpected way that keeps results from showing. I can't imagine what that might be though. Obviously the student data is still there, so I don't have a good lead on what is going on.

Hopefully they can provide more information when they come across it. It doesn't seem to be happening for every student. I would think that if it were a systemic issue, that would be the case.

**Comment by mquigley1** [comment]
Hi Craig,
I think I may have found part of, or a similar, issue related to #59.  I've attached a document with examples, and at the bottom, a name for you to try, or possibly a way for you to recreate the problem.  See attached. Thanks! 
[SMSK Not enrolled in this program Issue.docx](https://github.com/SMSK-CO/SMSK/files/1820782/SMSK.Not.enrolled.in.this.program.Issue.docx)
Michelle

**Comment by craigmcampbell** [comment]
Thanks for sending this over! This is such an odd issue. I tried the search that you suggested and was able to find the student without trouble. I'll try your suggestion on enrolling a new student.

**Comment by mquigley1** [comment]
You're welcome.  Yes, it is... I hope you can recreate the issue!


Sent from my T-Mobile 4G LTE Device

-------- Original message --------

**Comment by craigmcampbell** [comment]
@thomasnatalie @mquigley1 I am still looking into this to try to track down the issue. Since I haven't heard of any more instances of this happening, is it safe to say this is happening sporadically or do you think there is just a lull in system usage lately?

**Comment by thomasnatalie** [comment]
Craig, are you able to re-create the issue? I haven't heard anything but I think that's because of a lull in usage, not because it's not happening. It does appear to be sporadic, but I am also hearing about it more frequently since the 1st of the year (also not sure if that is because people now know to report to me or if it's becoming more frequent). If I hear anything more, I will send it your way.

**Comment by craigmcampbell** [comment]
Nope, not yet. It works just fine for me. I've looked through the code and I can't find any reason why it would be doing this.

**Comment by craigmcampbell** [comment]
@mquigley1 @thomasnatalie Is this still an issue? I haven't heard anything about it in a while. Still haven't been able to duplicate.
